tox.ini

```diff
@@ -27,9 +28,9 @@ extras =
 commands =
   ruff format --check qiskit_addon_utils/ docs/ test/
   ruff check qiskit_addon_utils/ docs/ test/
+  ruff check --select D qiskit_addon_utils/
```

Comment:
I would prefer to run `ruff` a single time and to instead modify

```
[tool.ruff.lint]
select = [...]
```
in pyproject.toml.

I will demonstrate this with a commit to the cutting addon shortly.

Reply:
The issue is not wanting to run `D` on `tests/`. You end up with a lot of violations. Adding docstring to tests if annoying.

Reply:
Ah, I'll set up `[tool.ruff.lint.per-file-ignores]`. It takes globs.